src/tasks.js

```diff
@@ -0,0 +1,34 @@
+// tasks.js
+
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function addTask(description) {
+  const newTask = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
```

Comment:
- [ ] As per the requirement, the item is added to the list should have the index of the tasks' length. So if you add one item, its index should start from 1, not 0. When you are creating an item and set its index to tasks.length as you did above, since it's not saved yet, the tasks.length will return 0. Please fix it.
![image](https://github.com/Peter-Kibet/ToDo_List/assets/91486791/5c4672f8-1fe2-41e9-b5f7-ad356cdb6391)